pkg/pillar/diskmetrics/usage.go

```diff
@@ -14,9 +14,15 @@ import (
 // SizeFromDir performs a du -s
 func SizeFromDir(log *base.LogObject, dirname string) uint64 {
```

Comment:
Please add a comment that we avoid ioutil.ReadDir and filepath.Walk since they do expensive and unnecessary sorting

Reply:
Done.